build.gradle

```diff
@@ -354,6 +354,30 @@ project(':iceberg-aws') {
   }
 }
 
+project(':iceberg-gcp') {
+  dependencies {
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    api project(':iceberg-api')
+    implementation project(':iceberg-common')
+    implementation project(':iceberg-core')
+
+    implementation platform('com.google.cloud:libraries-bom:24.1.0')
```

Comment:
use `versions.props` for version number. Also if we use that, I think we don't need to load the bom?

Reply:
Good catch.  As for the bom, I'm not super clear on what all the bom provides, but I think it insures that all of the required dependencies for the google libraries stay in sync.  Given the set of dependencies required here, it might be best to adhere to the bom (unless we find significant conflict with the spark libs).